src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -0,0 +1,256 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import styles from "./HighRiskBreachLayout.module.scss";
+import CreditCardIllustration from "../images/high-risk-data-breach-credit-card.svg";
+import BankAccountIllustration from "../images/high-risk-data-breach-bank-account.svg";
+import pinIllustration from "../images/high-risk-data-breach-pin.svg";
+import SocialSecurityNumberIllustration from "../images/high-risk-data-breach-ssn.svg";
+import NoBreachesIllustration from "../images/high-risk-breaches-none.svg";
+import { GuidedExperienceBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import { SubscriberBreach } from "../../../../../../../../utils/subscriberBreaches";
+import { useL10n } from "../../../../../../../hooks/l10n";
+import { ResolutionContentLayout } from "../ResolutionContentLayout";
+
+type HighRiskBreachLayoutProps = {
+  typeOfBreach: "creditCard" | "ssnBreaches" | "bankAccount" | "pin" | "none";
+  breachData: GuidedExperienceBreaches;
+};
+
+export const HighRiskBreachLayout = (props: HighRiskBreachLayoutProps) => {
+  const l10n = useL10n();
+  const highRiskDataBreaches = props.breachData.highRisk;
+  let exposedData: SubscriberBreach[] = [];
+
+  if (props.breachData) {
+    switch (props.typeOfBreach) {
+      case "ssnBreaches":
+        exposedData = highRiskDataBreaches.ssnBreaches;
+        break;
+      case "creditCard":
+        exposedData = highRiskDataBreaches.creditCardBreaches;
+        break;
+      case "bankAccount":
+        exposedData = highRiskDataBreaches.bankBreaches;
+        break;
+      case "pin":
+        exposedData = highRiskDataBreaches.pinBreaches;
+        break;
+      default:
+        "none";
+        break;
+    }
+  }
+
+  // TODO: Make locale location-sensitive in the future
+  const dateFormatter = new Intl.DateTimeFormat("en-US", {
+    // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#datestyle
+    dateStyle: "short",
+  });
+
+  const listOfBreaches = exposedData.map((item: SubscriberBreach) => (
+    <div key={item.id} className={styles.breachItem}>
+      {/* TODO: Localize all date values MNTOR-2111 */}
```

Comment:
I think you added this above? :)

```suggestion
      {/* TODO: Localize all date values MNTOR-2111 */}
```